.github/workflows/blackduck.yml

```diff
@@ -0,0 +1,45 @@
+name: blackduck-scan
+
+on:
+  pull_request: ~
+  push:
+    branches: [2.0]
```

Comment:
Should we add a comment or follow up ticket to change this to `main` once we merge 2.0 into it?

Reply:
Yeah good point, perhaps we should add it to whatever ticket we have to switch the default branch back to main.